src/main/java/lotto/model/LottoRank.java

```diff
@@ -0,0 +1,36 @@
+package lotto.model;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+    FIRST(6, 200_0000_000),
+    SECOND(5, 1_500_000),
+    THIRD(4, 50_000),
+    FOURTH(3, 5_000),
+    FIFTH(2, 0),
+    SIXTH(1, 0),
+    SEVENTH(0, 0);
+
+    private final int hits;
+    private final int cashPrize;
+
+    LottoRank(int hits, int cashPrize) {
+        this.hits = hits;
+        this.cashPrize = cashPrize;
+    }
+
+    public int getHits() {
+        return hits;
+    }
+
+    public int getCashPrize() {
+        return cashPrize;
+    }
+
+    public static LottoRank findByHits(int hits) {
+        return Arrays.stream(LottoRank.values())
+                .filter(lottoRank -> lottoRank.hits == hits)
+                .findAny()
+                .orElseThrow(() -> new IllegalStateException("로또 등수의 범위가 아닙니다."));
```

Comment:
FIFTH(2, 0), SIXTH(1, 0), SEVENTH(0, 0) 대신 MISS(0,0)  로 변경한다면 
```suggestion
        return Arrays.stream(LottoRank.values())
                .filter(lottoRank -> lottoRank.hits == hits)
                .findAny()
                .orElse(MISS)
```
와 같이 에러 없이 처리 할 수 있을 것 같아요.

7이상의 숫자가 들어올수는 있지만 이는 LottoRank에서 검증해야할 부분은 아닌 것 같아요.

Reply:
ENUM한테 에러까지 너무 많은 부분을 담당하게 하였네요.
2개 이하는 MISS로 처리하는게 훨씬 좋은 방향인 것 같습니다.

감사합니다 : )